src/objects/token/permissions.py

```diff
@@ -43,6 +43,14 @@ def has_object_permission(self, request, view, obj):
         if not object_permission:
             return False
 
+        # forbid showing history if there is field based auth
```

Comment:
I think this is a good call for now.

Reply:
"history" andpoint has other structure of the response than other endpoints, so I thought this would be the easiest to support solution